src/services/index.js

```diff
@@ -18,8 +19,9 @@ module.exports = function () { // eslint-disable-line func-names
   mongoose.connect(app.get('mongodb'));
   mongoose.Promise = Promise;
 
-  app.configure(authentication);
+  app.configure(activityLog);
   app.configure(app3rd);
+  app.configure(authentication);
```

Comment:
I think there should be no side-effect in this case. The `app3rd` and `activityLog` services require `user` object from `authentication` service but, after the server runs, the authentication service is ready.

Anyway, it's a good practice to place `authentication` service at the top. I'll make it in a new re-ordering PR.